tensorflow/src/main/scala/me/shadaj/scalapy/tensorflow/keras/models/Sequential.scala

```diff
@@ -35,14 +35,14 @@ trait Sequential extends py.Object {
       .as[Unit]
 
   def fit(
-      x: NDArray[Long],
+      x: NDArray[Float],
       y: NDArray[Long],
```

Comment:
Is it intentional that `y` is `Long`s but  x is `Float`s? if so... maybe add a comment in doc for the ML laymen like myself ;)

Reply:
Ok my bad, we should use the wider numeric type for safety.
The imdb dataset is `x: Long, y: Long (1 or 0)`
The mnist dataset is `x: Long y: Long(0-9)`
But for example Linear regression can work on `Float -> Float`